### PR TITLE
Added some examples from the dorahacks workshop

### PR DESCRIPTION
**Pull request by scglenn**

### CODE REVIEW

File: bridges/nomad/test/nomad.t.sol [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+        /* ---- CREATE FORK AT PRE-EXPLOIT BLOCK ---- */
+        previousFork = vm.createSelectFork("mainnet", 15_259_100);
+        previousTrapContract = IDroseraTrap(_testDeployTrapContract());
+
+        /* ---- CREATE FORK AT POST-EXPLOIT BLOCK ---- */
+        newFork = vm.createSelectFork("mainnet", 15_259_101);
+        newTrapContract = IDroseraTrap(_testDeployTrapContract());
+
+        // /* ---- CREATE USERS ---- */
+        // users = ScenarioUtils.createUsers(10);
```
Comment (An0nJon) [inline bridges/nomad/test/nomad.t.sol:45]: do these comments need to be here?

File: bridges/nomad/README.md [hunk 2]
```diff
@@ -0,0 +1 @@
+
```
Comment (An0nJon) [inline bridges/nomad/README.md:1]: Do we need a small read me here?

**Review by An0nJon — CHANGES_REQUESTED** [review]

File: bridges/nomad/test/nomad.t.sol (hunk 1 above)
Comment (scglenn) [reply bridges/nomad/test/nomad.t.sol]: removed

File: bridges/nomad/README.md (hunk 2 above)
Comment (scglenn) [reply bridges/nomad/README.md:1]: Adding

**Review by An0nJon — APPROVED** [review]

**Pull request merged**